src/nvim/lua/executor.c

```diff
@@ -520,6 +520,15 @@ static int nlua_state_init(lua_State *const lstate) FUNC_ATTR_NONNULL_ALL
     }
   }
 
+  {
```

Comment:
do you plan to actually add this file or what?

Reply:
nope, reverted locally already.